concepts/Common/Licensing/15 License Key Expiration and How to Update.md

```diff
@@ -0,0 +1,5 @@
+A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 
```

Comment:
Do not mention beta - we've just decided that no key will be available for betas
```suggestion
A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, etc. When switching to a new major version, the license key must be updated. 
```